webtau-feature-testing/pom.xml

```diff
@@ -102,6 +102,25 @@
                 </configuration>
             </plugin>
 
+            <plugin>
```

Comment:
This is apparently the way to get a random free port with the springboot maven plugin: https://docs.spring.io/spring-boot/docs/current/maven-plugin/examples/it-random-port.html